src/components/Tag/Tag.tsx

```diff
@@ -89,13 +85,13 @@ export const DissmissibleTag = polyRef<"button", TagProps, OnlyAs<"button" | "a"
   size,
   ...props
 }, ref) => {
-  const _classes = cn('fr-tag--dismiss', getTagClasses({ className, color, icon, iconPosition, size }));
+  const _classes = cn('fr-tag--dismiss', 'custom-dismissible-tag', getTagClasses({ className, color, icon, iconPosition, size }));
```

Comment:
normalement vient remplacer fr-tag--dismiss

Reply:
Thanks, I had a doubt writing this